### ISSUE TITLE
Polybar

### ISSUE THREAD

**Issue by maplepy** [YoungFellow-le/photoshop-22-linux#19]
### A clear description of the bug:

Basically the window is fine if it is full-screen but the issue is that it doesnt stay full screen
whenever i click on the window, it will show polybar back and even if I kill polybar, that still won't fix it

Is there something to do with wine to force the app to be fullscreened? I don't have any issue with other applications I run through wine like Overwatch or Fall guys for instance, they stay fullscreen

### The output from the installer:
```
N/A
```

### Screenshots:

![image](https://user-images.githubusercontent.com/28269333/214450685-db28e645-7c7c-447e-8866-7255aeeae319.png)
![image](https://user-images.githubusercontent.com/28269333/214450767-192963f7-1022-4a85-80da-92ff9bf815e8.png)
### System info:
 - **Linux distrobution:6.1.7-arch1-1**
 - **Desktop environment:BSPWM**
 - **Wine version:wine-7.22**

**Comment by YoungFellow-le** [comment]
Hello there.
Yes, Photoshop doesn't use the conventional window methods used by most Windows applications which makes it a bit unstable.
One workaround I found was to not maximise the window and only resize it until it fills the screen.
The other thing that fixes this issue for me is to tell KDE to force the window to use window borders and window titles

**Comment by maplepy** [comment]
> One workaround I found was to not maximise the window and only resize it until it fills the screen

I did not click to maximize, it automatically fits the whole screen, including the polybar space, I will try to click the maximize button again to not maximize it

> I'm not sure if that option is available in BSPWM.
Yeah I'm not sure it is either, I will have a look tonight but so far the best way of handling it I found was to set a virtual desktop size to my resolution minus the polybar size which kinda works ok

**Comment by YoungFellow-le** [comment]
One other thing you can try is to drag the window using the hot key.
It's usually `alt` + drag
Or `Super/Win` + drag

**Comment by maplepy** [comment]
Yeah, I've tried setting it to floating to drag it but for some reason it kept on moving back, will try to show you the issue with floating mode later too
